pkg/signal/signal_common.go

```diff
@@ -71,3 +71,11 @@ func ParseSysSignalToName(s syscall.Signal) (string, error) {
 	}
 	return "", fmt.Errorf("unknown syscall signal: %s", s)
 }
+
+func ToDockerFormat(s uint) string {
+	var signalStr, err = ParseSysSignalToName(syscall.Signal(s))
+	if err != nil {
+		signalStr = strconv.FormatUint(uint64(s), 10)
+	}
+	return fmt.Sprintf("SIG%s", signalStr)
```

Comment:
this looks odd, in case of an error it would return SIG2 for example. Is this really what docker does? To me it the more logical thing would be to return just the number without the prefix.

Reply:
I don't know how docker handle  unknown signal. I change this to return only the number as you ask.